controller/v1/base/home.go

```diff
@@ -0,0 +1,19 @@
+package base
```

Comment:
Like you did with the `home.go` file inside the routes, I would expect that you would have done the same thing with the controller here...move it out of `v1`